src/Helpers/AsyncPipe.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe } from 'riteway'
+import { asyncPipe } from './AsyncPipe'
+
+describe('asyncPipe', async (should: any) => {
```

Comment:
Let's remove this unit test since we're adding rambda today

Reply:
ok, I'm going to remove it now